src/components/http/Response.js

```diff
@@ -112,7 +113,8 @@ class Response {
             );
 
         // Match status code Number to a status message and call uWS.Response.writeStatus
-        this.#status_code = code + ' ' + status_codes[code];
+        this.#status = code + ' ' + status_codes[code];
```

Comment:
For this, instead of using two properties, It would be better to keep the only #status_code property but just change up the functionality such that the Response.status(code) method simply stores the provided Number into the #status_code property. Since HyperExpress, only needs to append the HTTP Code message during the uWS.Response.writeStatus() call which is done in Response._initiate_response(), we can simply retrieve and append the appropriate status code message for the #status_code at that time.

Reply:
Pushed this fix.